gpu_ocean/SWESimulators/NetCDFInitialization.py

```diff
@@ -238,7 +243,7 @@ def getInitialConditions(source_url, x0, x1, y0, y1, timestep_indices=None, land
     ic['t0'] = t0
     ic['timesteps'] = timesteps
     
-    #Number of cells
+    #Number of cells - FIXME: this is really really misleading!
```

Comment:
Not clear what "this" is..?

Reply:
sponge cells below in the pre-fix state (se previous comment about sponge cells)

Reply:
I still don't get why something here is really really misleading... :| Does that mean that I am mislead? Is it the meaning of NX vs nx? 

Since we agreed that sponge cells are part of the interior (with the exception of two of them being ghost cells), I don't see what is misleading. And therefore, the comment becomes very confusing

Reply:
The use of sponge_cells is really error prone (see previous comments). I agree that this comment is not helpful at the moment, and can remove it if that's what it takes.